src/components/structures/GroupView.js

```diff
@@ -670,7 +677,7 @@ export default React.createClass({
                 <h3>{ _t('Rooms') }</h3>
                 { addRoomRow }
             </div>
-            <RoomDetailList rooms={this._groupStore.getGroupRooms()} />
+            <RoomDetailList rooms={this.state.groupRooms} loading={this.state.groupRoomsLoading} />
```

Comment:
Is there a reason for passing this into `RoomDetailList` and letting it deal with the loading state rather than just showing a spinner in lieu of it?

Reply:
Not really, TBH. Happy do to the simpler thing of replacing it entirely with a spinner.